src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService.cs

```diff
@@ -44,6 +44,9 @@ protected override bool IsInNonFirstQueryClause(ExpressionSyntax expression)
             return false;
         }
 
+        protected override bool IsInterpolatedStringExpression(ExpressionSyntax expression)
+            => expression.IsKind(SyntaxKind.InterpolatedStringExpression);
```

Comment:
can be done without an abstrct/override.  just get teh ISyntaxKindsService and check the kind of the node against syntaxKinds.InterpolatedStringExpression